stories/examples/RangeSlider.js

```diff
@@ -24,6 +38,52 @@ const wrapper = (props = {}) => {
   )
 }
 
+export class DisplayTextSlider extends Component {
+  static propTypes = {
+    minValue: PropTypes.number,
+    maxValue: PropTypes.number,
+    theme: PropTypes.object,
+    value: PropTypes.object,
+  }
+
+  constructor(props) {
+    super(props)
+    this.onChange = this.onChange.bind(this)
+    this.state = { value: props.value}
+  }
+
+  onChange(value) {
+    this.setState( {value} )
+  }
+
+  render() {
+    const  {
+      minValue,
+      maxValue,
+      theme,
+      value,
+      ...props
+    } = this.props
+
+    console.log('currentval:', this.state.value)
+    return (
+    <div>
+    <h2 className={theme.textCenter}>Square Feet</h2>
+    <h2 className={theme.textCenter}>{this.state.value.min} ft - {this.state.value.max} ft</h2>
+    <RangeSlider
+      {...props}
```

Comment:
please run `yarn run lint:fix` and format the indentation of your html, use https://prettier.io/ if you do not have an easy way to do this in your editor